includes/class-wc-payments.php

```diff
@@ -1673,6 +1674,36 @@ public static function enqueue_dev_runtime_scripts() {
 		}
 	}
 
+	/**
+	 * Adds WCPay options to Woo Core option allow list.
+	 *
+	 * @param   array $permissions Array containing the permissions.
+	 *
+	 * @return  array              An array containing the modified permissions.
+	 */
+	public static function add_wcpay_options_to_woocommerce_permissions_list( $permissions ) {
+		$wcpay_permissions_list = [
```

Comment:
What do you think about doing something like this?

```
$wcpay_permissions_list = array_fill_keys(
			[
				'wcpay_frt_discover_banner_settings',
				'wcpay_multi_currency_setup_completed',
				'woocommerce_dismissed_todo_tasks',
				'woocommerce_remind_me_later_todo_tasks',
				'woocommerce_deleted_todo_tasks',
				'wcpay_fraud_protection_welcome_tour_dismissed',
				'wcpay_capability_request_dismissed_notices',
				'wcpay_onboarding_eligibility_modal_dismissed',
			],
			true
		);
```

Reply:
I have the `array_fill_keys` in the conditional below. Would you want to move it up into the `$wcpay_permissions_list` variable assignment so we only call it once?